rollup.config.js

```diff
@@ -29,28 +30,39 @@ export default [
       resolve(),
       sass({
         insert: true,
-        processor: css => postcss([autoprefixer, cssnano])
-          .process(css)
-          .then(result => result.css),
+        processor: css =>
+          postcss([autoprefixer, cssnano])
+            .process(css)
+            .then(result => result.css),
       }),
       uglify(),
     ],
   },
   {
-    input: 'test/es6-import-test.js',
+    input: 'src/index.js',
     output: {
-      file: 'dist/es6-import-test.js',
-      format: 'umd',
-      name: 'es6-import-test',
+      file: pkg.main,
+      format: 'esm',
+      name: 'canonicalGlobalNav',
+      sourcemap: true,
     },
     plugins: [
       babel({
         babelrc: false,
-        exclude: ['dist/**'],
+        exclude: ['node_modules/**', 'src/**/*.scss'],
         presets: [['env', { modules: false }], 'stage-0'],
         plugins: ['external-helpers'],
       }),
       commonjs(),
+      resolve(),
+      sass({
+        insert: true,
+        processor: css =>
+          postcss([autoprefixer, cssnano])
+            .process(css)
+            .then(result => result.css),
+      }),
+      terser(),
```

Comment:
Assuming this output format is meant to be consumed by a project build system does it need this step at all? It will likely be bundled, transpiled, etc separately anyway?

Reply:
Currently our sites `cp` the built files from this package. We could introduce a bundler for our sites and import the source but we need to support the basic usage for the time being.

Reply:
Yes, but as far as I understand what we want to copy is IIFE format (that is bundled separately and uglified).
This one (ESM) is meant to be imported by apps and bundled by project using it, right?

Reply:
@b-m-f For me it seems unnecessary. It adds another dependency to the project, but it doesn't seem to give any benefit if ESM module is going to be imported and bundled separately by outside project anyway.

Reply:
I have removed it :+1: